packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenuItemWrapper/ContextualMenuItemWrapper.tsx

```diff
@@ -3,7 +3,7 @@ import { BaseComponent, shallowCompare } from '../../../Utilities';
 import { IContextualMenuItemWrapperProps } from './ContextualMenuItemWrapper.types';
 import { IContextualMenuItem } from '../../../ContextualMenu';
 
-export class ContextualMenuItemWrapper extends BaseComponent<IContextualMenuItemWrapperProps, {}> {
+export class ContextualMenuItemWrapper extends BaseComponent<IContextualMenuItemWrapperProps> {
```

Comment:
While you're at improving things, can this just extend React.PureComponent rather than BaseComponent?

Reply:
That change can have unintended consequences that I'm not prepared to address as part of this PR. I like to have easily revertible PRs that encapsulate units of work in the case of something going awry.

It is something that should be pursued though. If not `PureComponent` then at least `React.Component`.

I honestly don't see the value in keeping `ContextualMenuItemWrapper` long term given my experience with it.